t/t1036-mf-priority-resource-counting.t

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+test_description='test calculating and storing system core information in priority plugin'
+
+. `dirname $0`/sharness.sh
+
+mkdir -p conf.d
+
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+SUBMIT_AS=${SHARNESS_TEST_SRCDIR}/scripts/submit_as.py
+DB_PATH=$(pwd)/FluxAccountingTest.db
+
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 4 job -o,--config-path=$(pwd)/conf.d
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'load multi-factor priority plugin' '
+	flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
+'
+
+test_expect_success 'check that mf_priority plugin is loaded' '
+	flux jobtap list | grep mf_priority
+'
+
+test_expect_success 'check that cores-per-node count is correct' '
+	flux jobtap query mf_priority.so > query.json &&
+	test_debug "jq -S . <query.json" &&
+	cat query.json &&
+	jq -e ".ncores_per_node == 2" <query.json
```

Comment:
Will this check work on all systems? If you need to check the actual cores/node you could use `nproc` or possibly `flux resource R --include=0 | flux R decode --count=core` (sorry if I missed someplace where you've guaranteed 2 cores per node)